### PR TITLE
Open composer from an email address in message view

### PR DESCRIPTION
**Pull request by DeepDiver1975**
TODO
* [ ] ~~implement adding to Contacts~~
* [ ] ~~only show »Add to Contacts« button for email addresses which are not in the contacts yet~~
* [x] for »Send message to«, prefill not only email but also the name if it was there

### CODE REVIEW

File: js/mail.js [hunk 1]
```diff
@@ -265,7 +265,10 @@ var Mail = {
 					if (currentAddress && email === currentAddress.emailAddress) {
 						label = t('mail', 'you');
 					}
-					return memo + '<span title="' + email + '">' + label + '</span>';
+					var title = t('mail', 'Send message to {email}', {email: email});
+					memo += '<span class="tipsy-mailto" title="' + title + '"><a class="link-mailto" data-email="' + email + '">' + label + '</a></span>';
```
Comment (jancborchardt) [inline js/mail.js]: Instead of wrapping it in a span, you can also just add the `tipsy-mailto` class and the title to the `a` link element. ;)

**Comment by jancborchardt** [comment]
Cool stuff! Just a nitpick in the code, and I added a TODO to the original issue.

File: js/mail.js (hunk 1 above)
Comment (DeepDiver1975) [inline js/mail.js]: I guess I tried that and the styling was broken - only spans are properly aligned afaik ...

**Comment by DeepDiver1975** [comment]
@jancborchardt let's roll this in - I'll implement adding contacts in a second pr

**Comment by jancborchardt** [comment]
Ok. second review please @irgendwie @aidanamavi @zinks- everyone. :)

File: js/mail.js [hunk 2]
```diff
@@ -587,6 +591,10 @@ var Mail = {
 
 			// focus 'to' field automatically on clicking New message button
 			$('#to').focus();
+			if (!_.isUndefined($(data.currentTarget).data().email)) {
```
Comment (irgendwie) [inline js/mail.js]: if I open a mail in the **drafts** folder this happens:
`Uncaught TypeError: Cannot read property 'email' of null`

Seems like `data.currentTarget` doesn't exists there and the desired information are in data itself.

@DeepDiver1975

Comment (aidanamavi) [inline js/mail.js]: I get a similar error in Firefox:
TypeError: $(...).data(...) is null          mail.js:594:22

**Comment by DeepDiver1975** [comment]
Please review and test @jancborchardt @irgendwie @aidanamavi @zinks- or anybodyelse - THX

**Comment by aidanamavi** [comment]
:+1: Works well.

**Comment by DeepDiver1975** [comment]
Thanks Aiden for testing!

**Pull request merged**